### PR TITLE
start work on deepin in Release/next

### PR DESCRIPTION
**Pull request by pieckenst**
Basic start for deepin build files , helps out with #65

### CODE REVIEW

File: src/cores/deepin.yaml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+include: [
+  "./common.yaml",
+  "../packages/deepin.yaml"
```
Comment (pieckenst) [inline src/cores/deepin.yaml:3]: took gnome file as base , likely would need edits later on - but good for start point

File: src/packages/deepin.yaml [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+ - deepin-system-monitor
+ - deepin-account-faces
+ - deepin-gettext-tools #(not updated in long term)
+ - deepin-gtk-theme #(not updated in long term)
+ - deepin-sound-theme
+ - deepin-shortcut-viewer
+ - deepin-screensaver
+ - deepin-calendar
+ - deepin-draw
+ - deepin-kwin
```
Comment (pieckenst) [inline src/packages/deepin.yaml:45]: just everything listed on https://fedoraproject.org/wiki/SIGs/DeepinDE

**Comment by electricduck** [comment]
This belongs in the branch `main`. We shouldn't really be making PRs to the release branches 😃.

**Comment by pieckenst** [comment]
alright ill switch it over to main:)

**Comment by electricduck** [comment]
Manually merged in 83052174b59cdf0009e68687725f5e5e8a556c58

**Pull request closed without merging**